src/main.ts

```diff
@@ -314,8 +308,8 @@ export default class NewZettel extends Plugin {
 
     if (placeCursorAtStartOfContent) {
       let line = 2
-      if (this.settings.addAlias && this.settings.useLinkAlias) {
-        line = 6
+      if (this.settings.addAlias) {
```

Comment:
I am not totally familure with this area of the code but I am trusting you here for now if you could drop a line telling why `this.settings.usLinkAlias` is nolonger needed here that would be great. ( I will be murgeing the code so just comment on the closed PR please.

Reply:
this line is to account for the frontmatter in the document, ie. "start of content" is no longer line 2, but with frontmatter is +4 lines lower.

Original implementation had both settings required, but in previous commit updated so they can be set independently, so only needed to check addAlias